contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
Fix for new `st2ctl`: https://github.com/StackStorm/st2/pull/2385 to use `systemctl` when possible.